intelligence/serializers.py

```
@@ -144,3 +143,29 @@ def get_summary_preview(self, obj):
         """Return count of linked clinical references"""
         return obj.references.count()
+
+
+# Simple serializer for the API gateway (backward compatibility)
+class AISummarySimpleSerializer(serializers.ModelSerializer):
+    resource_type = serializers.CharField(read_only=True)
+    resource_id = serializers.SerializerMethodField()
+    
+    class Meta:
+        model = AISummary
+        fields = ['id', 'resource_type', 'resource_id', 'summary', 'created_at']
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)